enforcer-rules/src/it/fail-build-for-linkage-errors/verify.groovy

```diff
@@ -0,0 +1,8 @@
+def buildLog = new File(basedir, "build.log")

```

Comment:
This is groovy script specified by ["postBuildHookScript"](http://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#postBuildHookScript).

Maven-checkstyle-plugin also uses groovy script to verify the build https://github.com/apache/maven-checkstyle-plugin/tree/master/src/it/MCHECKSTYLE-338